tstate/tstate_test.go

```diff
@@ -663,60 +665,42 @@ func TestGetValuePermissions(t *testing.T) {
 			ctx := context.TODO()
 			ts := New(10)
 			tsv := ts.NewView(state.Keys{tt.key: tt.permission}, map[string][]byte{tt.key: testVal})
-			if !tt.shouldErr {
-				_, err := tsv.GetValue(ctx, []byte(tt.key))
-				require.NoError(err)
-			} else {
-				_, err := tsv.GetValue(ctx, []byte(tt.key))
-				require.ErrorIs(ErrInvalidKeyOrPermission, err)
-			}
-
-			require.Equal(tt.canRead, tt.permission.Read)
-			require.Equal(tt.canWrite, tt.permission.Write)
```

Comment:
I felt like these checks were a bit weird because they were testing constants from the test struct directly... If these were supposed to be testing other things lmk

Reply:
That's a good point, before the `state/keys_test.go` I kinda intended to use that to check if `permission` was being properly set